controllers/user.js

```diff
@@ -761,6 +761,27 @@ async function getRemovalEnrolledPage(req, res) {
   });
 }
 
+async function handleRemovalOptout(req, res) {
+  const sessionUser = req.user;
+  const optoutRes = await DB.removalOptout(sessionUser);
+  if (!optoutRes || optoutRes !== 1) {
+    const localeError = LocaleUtils.formatRemoveString("remove-error-optout");
+    return res.status(400).json({
+      error: localeError,
+    });
+  }
+
+  if (!req.session || !req.session.kanary) {
+    const localeError = LocaleUtils.formatRemoveString("remove-error-optout");
+    return res.status(400).json({
+      error: localeError,
+    });
+  }
+
```

Comment:
Should these two `if` statements be combined?

Reply:
ah - no, but good catch - the error response printed out in the second is incorrect.